src/coreclr/src/jit/codegenlinear.cpp

```diff
@@ -938,6 +938,84 @@ GenTree* sameRegAsDst(GenTree* tree, GenTree*& other /*out*/)
     }
 }
 
+//------------------------------------------------------------------------
+// genUnspillLocal: Reload a register candidate local into a register, if needed.
```

Comment:
> `, if needed.`

Does this method decide if it is needed or not?

Reply:
Ah, no it doesn't. The caller has to determine that. I'll change the comment.